doc/examples/reconst_dki.py

```diff
@@ -189,38 +180,38 @@
 
 """
 The DT based measured obtain from DKI and DTI can be easly visualized using
-matplotlib. For example, we show below the middle axial slices of FA, MD, AD,
-and RD obtain from the DKI model (upper panels) and the DTI model (lower
-panels).
+matplotlib. For example, the FA, MD, AD, and RD obtain from the DKI model
+(upper panels) and the DTI model (lower panels) are ploted for an arbitary
```

Comment:
=> "are plotted for an axial slice."

"Plotted" has two t's and also no need for the "arbitrary selected"